### ISSUE TITLE
support for --files-from?

### ISSUE THREAD

**Issue by ginabythebay** [erikw/restic-systemd-automatic-backup#44]
Do you have any interested in supporting the --files-from option for the backup command?  I need something like that to just include certain files from a path I want to backup.  If it sounds interesting to you, I could try putting together a PR (my bash-fu is a little weak compared to yours, so any guidance would be helpful).

Also, thanks for putting this together, I think this is going to work great for me.

**Comment by erikw** [comment]
Hey,

I've not used the `--files-from` way myself but I can see how it can be useful. For this repo, I think it might be a bit too specific, as I intent that this repo is more general "tutorial" style to inspire people to make the full solution they need themselves.

However if you have an example of how you set this up for yourselves, you could share it here, and I could include a link in the README for those who are interested in this way of using restic :)

**Comment by ginabythebay** [comment]
> ...if you have an example of how you set this up for yourselves, you could share it here...

Sure....my use case is that I keep a lot of pdfs in google drive and want to back them up, but not all the other stuff that is in there.  They are accessible through fuse in /home/gina/gdrive.  My solution was to create the file `/etc/restic/backup_files_from` containing one line:

`/home/gina/gdrive/**/*.pdf`

Then I modified restic_backup.sh, adding this just before `BACKUP_TAG` is set:

```
BACKUP_FILES_FROM=""
[ -f /etc/restic/backup_files_from ] && BACKUP_FILES_FROM+="--files-from /etc/restic/backup_files_from"
```
and I modified the invocation of `restic backup`, adding 

```
        $BACKUP_FILES_FROM \
```
after the `BACKUP_EXCLUDES` part.
